Framework/PythonInterface/mantid/py3compat/__init__.py

```diff
@@ -37,6 +38,21 @@
 # -----------------------------------------------------------------------------
 __all__ = dir(six)
 
+VERSION_INFO = sys.version_info
+MAJOR_VER = VERSION_INFO[0]
+MINOR_VER = VERSION_INFO[1]
+
+
+# -----------------------------------------------------------------------------
+# System functions
+# -----------------------------------------------------------------------------
+if MAJOR_VER == 2:
```

Comment:
Can I suggest using `six` for the version check to avoid duplicating that code? Also, if the future name will be `setswitchinterval` could we use that as our compatability name instead of the deprecated name?

```python
if PY2 or sys.version_info[0:2] < (3, 2):
   setwitchinterval = sys.setcheckinterval
else:
    setwitchinterval = sys.setswitchinterval
```